pkg/storageincentives/agent.go

```diff
@@ -109,6 +104,7 @@ func (a *Agent) start(blockTime time.Duration, blocksPerRound, blocksPerPhase ui
 		phaseEvents    = newEvents()
 	)
 
+	a.nodeState.SetState(idle)
```

Comment:
not sure if we need this

Reply:
It was mentioned in issue so included this.